pkg/store/store.go

```diff
@@ -119,13 +122,31 @@ func (store *OpenTSDBStore) Info(
 		Labels:  store.storeLabels,
 	}
 	var err error
-	store.timedTSDBOp("version", func() error {
-		_, err = store.openTSDBClient.WithContext(ctx).Version()
+	store.timedTSDBOp("query", func() error {
+		subqueries := make([]opentsdb.SubQuery, 1)
```

Comment:
cleaner as:

```
subqueries := []opentsdb.SubQuery{{
  Metric: ...
}}
```

Maybe also put that straight into QueryParam without the temp depending how it looks.

Reply:
Agreed